packages/calcite-components/conventions/Styling.md

```diff
@@ -41,50 +43,63 @@ In the [global CSS file](https://github.com/Esri/calcite-design-system/blob/main
 </div>
 ```
 
-This will cause both the button and the date picker to use the dark mode color variables declared in the global file. This makes it very easy for developers to move an entire app from light to dark mode and vice versa.
-
 To make this work, inside a component's SASS file, _you must use colors from the theme variables_. For example
 
 ```scss
-// 🙅‍♀️ using the sass var will not correctly inherit or change in light/dark mode
+// 👍 Using the CSS var will inherit the value correctly
 :host {
-  color: $ui-brand-light;
+  color: var(--calcite-color-brand);
 }
+```
+
+```scss
+@import "~@esri/calcite-design-tokens/dist/scss/light";
 
-// 👍 using the CSS var will inherit correctly
+// Using the sass var will set the color of host to the value of the brand color in light mode
+// 🙅‍♀️ However, it will not correctly inherit or change it's value when swapping light/dark mode
 :host {
-  color: var(--calcite-color-brand);
+  color: $calcite-color-brand;
 }
 ```
 
+## Legacy Tokens
+
+In the release of 2.0 Calcite Component styles got a major refactor which included the removal and reassignment of legacy CSS Custom Properties originally introduced through calcite-styles/calcite-colors. To see a full list of CSS Custom Property additions, deletions, and renamed tokens please refer to the [Calcite Design Tokens 2.0 Changelog > Map of token changes](../../calcite-design-tokens/CHANGELOG.md#20-map-of-token-changes).
```

Comment:
Can the first sentence be copyedited a bit, similar to the [recommendation in the initial review](https://github.com/Esri/calcite-design-system/pull/8357#discussion_r1418067637)?